### DISCUSSION TITLE
GPX track and Route line width

### DISCUSSION THREAD

**Discussion by Petak73** [organicmaps/organicmaps#8524] [Ideas]
Since Organic Maps does not navigate along an imported GPX track, I use it by importing a GPX track and then asking Organic Maps to navigate to the end of the track. Obviously, the route made by OM does not coincide with the track and for longer tracks adding stops in order to make them coincide is annoying. Therefore, whenever the route deviates from the track, I simply follow the track until the route is recalculated. That would be much easier to do if the GPX track line can be made (within OM) to be a bit wider and of a different colour than the blue route line. The route line in itself is a bit too wide - it should be equal in width to the road beneath it, IMO.

Based on the above, I would suggest:
- allow adjusting the GPX track width and colour in OM (for Android)
- reducing the width of the route line (for Android)

Providing an option to remove all POI from the map as well as the street name on top of the navigation screen will also help readability at a glance.

**Comment by biodranik** [comment]
It is a fight with symptoms instead of fixing the root cause. A proper track navigation should be implemented. See #1360

**Reply by Petak73** [reply]
"Proper track navigation" complicates things quite a lot, which seems to go against the idea behind Organic Maps. Having multicolour tracks clearly displayed with a route line on top in order to have the perspective and auto zoom working, seems good enough to me. Whenever the route line and the track split and go in different directions, you have a choice to follow either of them and you always have a reference to the original UNMODIFIED track, which is not the case with apps that recalculate the imported track in order to provide navigation.
